openstef/tasks/calculate_kpi.py

```diff
@@ -89,7 +119,12 @@ def check_kpi_pj(pj, context, start_time, end_time):
         context.database.ktp_api.add_tracy_job(pj["id"], function=function_name)
 
 
-def calc_kpi_for_specific_pid(pid, start_time=None, end_time=None):
+def calc_kpi_for_specific_pj(
+    pj: PredictionJobDataClass,
```

Comment:
We only seem to need the prediction job id. So maybe we don't give the whole prediction job object, but just the pid?

Reply:
yes, I changed it to pid